pkgs/stdenv/darwin/default.nix

```diff
@@ -253,43 +246,31 @@ rec {
           '';
         };
 
-        sigtool = stdenv.mkDerivation {
-          name = "bootstrap-stage0-sigtool";
-          phases = [ "installPhase" ];
-          installPhase = ''
-            mkdir -p $out/bin
-            ln -s ${bootstrapTools}/bin/sigtool $out/bin
-
-            # Rewrite nuked references
-            sed -e "s|[^( ]*\bsigtool\b|$out/bin/sigtool|g" \
-              ${bootstrapTools}/bin/codesign > $out/bin/codesign
-            chmod a+x $out/bin/codesign
-          '';
-          # on next bootstrap tools update, use the following:
-          # installPhase = ''
-          #   mkdir -p $out/bin
-          #   ln -s ${bootstrapTools}/bin/sigtool  $out/bin
-          #   ln -s ${bootstrapTools}/bin/codesign $out/bin
-          # '';
-        };
+        sigtool = self.runCommandLocal "bootstrap-stage0-sigtool" { } ''
+          mkdir -p $out/bin
+          ln -s ${bootstrapTools}/bin/sigtool $out/bin
 
-        print-reexports = stdenv.mkDerivation {
-          name = "bootstrap-stage0-print-reexports";
-          phases = [ "installPhase" ];
-          installPhase = ''
-            mkdir -p $out/bin
-            ln -s ${bootstrapTools}/bin/print-reexports $out/bin
-          '';
-        };
+          # Rewrite nuked references
+          sed -e "s|[^( ]*\bsigtool\b|$out/bin/sigtool|g" \
+            ${bootstrapTools}/bin/codesign > $out/bin/codesign
+          chmod a+x $out/bin/codesign
+        '';
+        # on next bootstrap tools update, use the following:
```

Comment:
If we're gonna update the bootstrap-tools archive we should probably do this.

Reply:
done.